modules/810-deckhouse-web/docs/USAGE.md

```diff
@@ -4,6 +4,8 @@ title: "The deckhouse-web module: usage"
 
 ## An example of the module configuration
 
+Below is a simple example of the module configuration:
```

Comment:
@Zhbert no need to repeat the same text I think. The header contains the same text.

Reply:
@z9r5 Maybe you're right. But I've always thought that a headline is a headline. After it, you need some kind of introduction. And the black square just behind the title looks ugly :)